### PR TITLE
EP4 - Model training and validation

### PR DESCRIPTION
**Pull request by TFUGSP**

### CODE REVIEW

**Comment by piEsposito** [comment]
Tem um problema com a avaliação do modelo. Acho que não estamos incluindo o pré-processamento na hora e ele nos retorna 34% de acurácia. Vou dar uma olhada.

**Review by piEsposito — CHANGES_REQUESTED** [review]
Podemos remover o componente Transform e colocar uma [camada de normalização](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Normalization) direto no modelo?

Além disso, tá como EP4. A gente pulou o 3 ou foi isso mesmo?

**Comment by piEsposito** [comment]
Podemos adicionar uma badge direta pro colab no readme pra deixar mais bonitinho? É como esse aqui abaixo, mas alterando o link (e depois de liberar no próprio colab).

[Open In Colab](https://colab.research.google.com/github/tensorflow-sp/tfx-iniciantes-2022/blob/min/Ep4/TFX_Pipelines_Treinamento_e_validação_de_modelos.ipynb)

**Comment by pedrogengo** [comment]
> Podemos adicionar uma badge direta pro colab no readme pra deixar mais bonitinho? É como esse aqui abaixo, mas alterando o link (e depois de liberar no próprio colab).
> 
> [Open In Colab](https://colab.research.google.com/github/tensorflow-sp/tfx-iniciantes-2022/blob/min/Ep4/TFX_Pipelines_Treinamento_e_valida%C3%A7%C3%A3o_de_modelos.ipynb)

Adicionado!

**Comment by pedrogengo** [comment]
> Podemos remover o componente Transform e colocar uma [camada de normalização](https://www.tensorflow.org/api_docs/python/tf/keras/layers/Normalization) direto no modelo?
> 
> Além disso, tá como EP4. A gente pulou o 3 ou foi isso mesmo?

Não subi a aula 3, preciso fazer isso. E removi a normalização por hora, acho que podemos comentar mas acabei não colocando para reduzir o conteúdo, já está com muita coisa hahaha

**Review by piEsposito — APPROVED** [review]
podemos colocar a badge do colab direto no readme pra ajudar o pessoal?
